src/shared/constants/realms.ts

```diff
@@ -112,6 +112,26 @@ export function getRealmColor(realmId: string): string {
   return color
 }
 
+export function getRealmPosition(realmId: string): {
+  x: number
+  y: number
+  width: number
+  height: number
```

Comment:
Just a suggestion, maybe we could add a separate type for it (like `RealmMapPosition`), I think this pattern is used somewhere else and if not it may come useful in the future